### PR TITLE
*: refine statements doc

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Tiny refine `Statements Summary` doc.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@crazycs520 Could you please invite one or more tech reviewers?

File: dashboard/dashboard-statement-list.md [hunk 1]
```diff
@@ -70,6 +70,12 @@ SQL 语句分析页面所展示的所有数据都来自于 TiDB Statement 系统
 >
 > 数据收集周期和保留时间的值会影响内存占用，因此建议根据实际情况调整，保留时间不宜设置过大。
 
+### Others
+
+[tidb_stmt_summary_max_stmt_count](/system-variables.md#tidb_stmt_summary_max_stmt_count) 控制 Statement 系统表保存的 SQL 种类数量。当 SQL 种类超过该值时，会移除最近没有出现的 SQL。这些 SQL 将会被 `DIGEST` 为 `NULL` 的行数据统计。`DIGEST` 为 `NULL` 的行数据在 TiDB Dashboard SQL 语句分析列表页面中将会显示为 `Others`, 如下所示：
```
Comment (djshow832) [inline dashboard/dashboard-statement-list.md:75]:
```suggestion
[tidb_stmt_summary_max_stmt_count](/system-variables.md#tidb_stmt_summary_max_stmt_count) 控制 Statement Summary 系统表保存的 SQL 种类数量。当 SQL 种类超过该值时，会移除最近没有出现的 SQL。这些 SQL 将会被 `DIGEST` 为 `NULL` 的行数据统计。`DIGEST` 为 `NULL` 的行数据在 TiDB Dashboard SQL 语句分析列表页面中将会显示为 `Others`, 如下所示：
```

**Review by djshow832 — COMMENTED** [review]
Rest LGTM

**Review by djshow832 — APPROVED** [review]

**Comment by crazycs520** [comment]
@TomShawn PTAL

**Comment by crazycs520** [comment]
@TomShawn PTAL

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: dashboard/dashboard-statement-list.md [hunk 2]
```diff
@@ -70,6 +70,12 @@ SQL 语句分析页面所展示的所有数据都来自于 TiDB Statement 系统
 >
 > 数据收集周期和保留时间的值会影响内存占用，因此建议根据实际情况调整，保留时间不宜设置过大。
 
+### Others
+
+[`tidb_stmt_summary_max_stmt_count`](/system-variables.md#tidb_stmt_summary_max_stmt_count) 控制 Statement Summary 系统表保存的 SQL 种类数量。当 SQL 种类超过该值时，会移除最近没有出现的 SQL。这些 SQL 将会被 `DIGEST` 为 `NULL` 的行数据统计。`DIGEST` 为 `NULL` 的行数据在 TiDB Dashboard SQL 语句分析列表页面中将会显示为 `Others`，如下所示：
```
Comment (shichun-0415) [inline dashboard/dashboard-statement-list.md:75]:
```suggestion
[`tidb_stmt_summary_max_stmt_count`](/system-variables.md#tidb_stmt_summary_max_stmt_count-从-v40-版本开始引入) 控制 Statement Summary 系统表保存的 SQL 种类数量。当 SQL 种类超过该值时，会移除最近没有出现的 SQL。这些 SQL 将会被 `DIGEST` 为 `NULL` 的行数据统计。`DIGEST` 为 `NULL` 的行数据在 TiDB Dashboard SQL 语句分析列表页面中将会显示为 `Others`，如下所示：
```

**Pull request merged**